internal/command/machine/clone.go

```diff
@@ -31,7 +31,7 @@ func newClone() *cobra.Command {
 
 	cmd := command.New(usage, short, long, runMachineClone,
 		command.RequireSession,
-		command.LoadAppNameIfPresent,
+		command.RequireAppName,
```

Comment:
@catflydotio and @alichay you may want to review this.

Looks like a different UX considering https://github.com/superfly/flyctl/pull/1708

Reply:
Thanks for the heads up!

For the most part, this all seems fine to me. `secrets` is, correct me if i'm wrong, a command that operates on Apps rather than machines, so it has to take an app. it makes sense for `clone` to need an explicit app provided because it's creating a new machine in that app, and `list` can't infer anything.

Personally, I would like to see the `lease` commands behave like `restart`, where it infers the app from the first machine ID.

whether we settle on this explicit style or `restart`'s implicit style, though, we should probably agree on one style for commands that take machine ids. I just feel like inferring the app from the machine ID makes things easier for users :)